appium-provider/src/main/java/com/paypal/selion/appium/platform/grid/SeLionAppiumIOSDriver.java

```diff
@@ -162,13 +161,27 @@ public String getValue(WebElement webElement) {
         return value;
     }
 
-    @Override
-    public void rotate(DeviceRotation deviceRotation) {
-        //TODO
+    public void swipe(int startx, int starty, int endx, int endy) {
+        swipe(startx, starty, endx, endy, SWIPE_DURATION);
     }
 
-    @Override
-    public DeviceRotation rotation() {
-        return null;
+    
+    public void swipe(int startx, int starty, int endx, int endy, int duration) {
+        int xOffset = endx - startx;
+        int yOffset = endy - starty;
+        new TouchAction(this).press(startx, starty).waitAction(Duration.ofMillis(duration)).moveTo(xOffset, yOffset)
+                .release().perform();
     }
+
+    public void tap(int fingers, WebElement element, int duration) {
+        MultiTouchAction multiTouch = new MultiTouchAction(this);
+
+        for (int i = 0; i < fingers; i++) {
+            TouchAction tap = new TouchAction(this);
+            multiTouch.add(tap.press(element).waitAction(Duration.ofMillis(duration)).release());
+        }
+
+        multiTouch.perform();
+    }
+
```

Comment:
I still need to test this action before I can comment on it since I am not sure if it is multi finger or multi tap action.

Reply:
I believe these are all multi finger actions.